pkg/cmd/cli/describe/deployments.go

```diff
@@ -27,6 +27,20 @@ import (
 	imageapi "github.com/openshift/origin/pkg/image/api"
 )
 
+const (
+	// MaxDisplayDeployments is the number of deployments to show when describing
+	// deployment configuration.
+	// TODO: Use this constant when limiting the number of deployments displayed in
```

Comment:
Plug this in `oc status` while here. Also we don't need to export any of these constants.